packaging/suse/amazon-ecs-init.spec

```diff
@@ -0,0 +1,91 @@
+#
+# spec file for package amazon-ecs-init
+#
+# Copyright (c) 2015 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+%define short_name amazon-ecs
+Name:           amazon-ecs-init
+Version:        1.4.0.2
```

Comment:
I don't think you'd typically need to care about the difference from 1.4.0-1 to 1.4.0-2 as it's usually just spec file changes for the Amazon Linux AMI (e.g., changing the Requires for Docker when we have a new version to be released).  On the other hand, you have a good point.  If you'd prefer to leave it this way (and expect that it would usually end in .1 for you), then I will not consider this a blocker.